src/main/java/reposense/RepoSense.java

```diff
@@ -102,4 +104,9 @@ public static void main(String[] args) {
 
         return configs;
     }
+
+    private static String getVersion() {
+        String version = RepoSense.class.getPackage().getImplementationVersion();
+        return  version == null ? "v1.7" : version;
```

Comment:
Not sure if it will be null. It seems to be `Unspecified` when run without `-Pversion`.

Reply:
Instead of using `v1.7` is it possible to leave it as an empty string?

Reply:
And also broke the link.

Reply:
Maybe we can ask other's opinion of `default version` ?